operator/config/crd/kustomization.yaml

```diff
@@ -31,7 +31,7 @@ patchesStrategicMerge:
 # patches here are for enabling the CA injection for each CRD
 #- patches/cainjection_in_keptntaskdefinitions.yaml
 #- patches/cainjection_in_keptntasks.yaml
-- patches/cainjection_in_keptnapps.yaml
+# - patches/cainjection_in_keptnapps.yaml
```

Comment:
why this plsease ?

Reply:
cainjection is to inject certmanager we do not want this anymore